contrib/dmidecode/template.py

```diff
@@ -0,0 +1,14 @@
+pkgname = "dmidecode"
+pkgver = "3.5"
+pkgrel = 0
```

Comment:
i believe this should be `archs = ["x86_64"]`, no?

Reply:
aarch64 also has smbios/dmi afaik. so perhaps just those 2, not sure if riscv will ever support that either..

Reply:
hm, think riscv also has it

Reply:
from what i can tell any platform that implements EFI can optionally then provide the smbios tables for this to end up working, i guess

Reply:
these 3 then